api/types/role.go

```diff
@@ -1913,6 +1913,13 @@ func (r *RoleV6) GetLabelMatchers(rct RoleConditionType, kind string) (LabelMatc
 		return LabelMatchers{cond.WindowsDesktopLabels, cond.WindowsDesktopLabelsExpression}, nil
 	case KindUserGroup:
 		return LabelMatchers{cond.GroupLabels, cond.GroupLabelsExpression}, nil
+	case KindIdentityCenterAccount:
+		var matchers LabelMatchers
+		accounts := utils.Transform(cond.AccountAssignments, IdentityCenterAccountAssignment.GetAccount)
+		if len(accounts) > 0 {
+			matchers.Labels = Labels{IdentityCenterAccountLabel: utils.Deduplicate(accounts)}
+		}
+		return matchers, nil
```

Comment:
Do we really need a new Transform function and templating here?

TBH I've been staring at this for 10 minutes trying to understand what this is doing - wouldn't just having a simple `GetICAccounts()` function on the role conditions object that returns a list of accounts names from AccountAssignments be the equivalent?

```go
case KindIdentityCenterAccount:
    return LabelMatchers{
        Labels: Labels{
            IdentityCenterAccountLabel: utils.Deduplicate(cond.GetICAccounts()),
        }
    }
```